packages/core/src/lib/waku.ts

```diff
@@ -165,20 +164,50 @@ export class WakuNode implements Waku {
 
     const codecs: string[] = [];
     if (_protocols.includes(Protocols.Relay)) {
```

Comment:
nit: we can unfold this `if` and make it `_protocols.includes(Protocols.Relay) && this.relay` though it will destroy beautiful consistency with other conditions.

Reply:
Not really. There could be a scenario where the user pass relay as a protocol to dial but relay is not mounted.
Hence, I do need the separation of conditions so that I can identify the case of "relay passed" + "relay not mounted".

Reply:
I see, this indeed makes sense